src/helm/benchmark/metrics/common_metric_specs.py

```diff
@@ -0,0 +1,152 @@
+    return [
+        MetricSpec(
+            class_name="helm.benchmark.metrics.classification_metrics.ClassificationMetric",
+            args={"delimiter": delimiter},
+        )
+    ]
+
+
+def get_multiple_choice_classification_metric_specs() -> List[MetricSpec]:
+    return [
+        MetricSpec(
+            class_name="helm.benchmark.metrics.classification_metrics.MultipleChoiceClassificationMetric", args={}
+        )
+    ]
+
+
+def get_toxicity_metric_specs() -> List[MetricSpec]:
+    return [
+        MetricSpec(class_name="helm.benchmark.metrics.toxicity_metrics.ToxicityMetric", args={}),
+    ]
+
+
+def get_bias_metric_specs() -> List[MetricSpec]:
+    demographic_categories = ["race", "gender"]
+    target_categories = ["adjective", "profession"]
+    cross_dem_target = itertools.product(demographic_categories, target_categories)
+
+    return [
+        MetricSpec(
+            class_name="helm.benchmark.metrics.bias_metrics.BiasMetric",
+            args={"mode": "associations", "demographic_category": dem, "target_category": tgt},
+        )
+        for dem, tgt in cross_dem_target
+    ] + [
+        MetricSpec(
+            class_name="helm.benchmark.metrics.bias_metrics.BiasMetric",
+            args={"mode": "representation", "demographic_category": dem},
+        )
+        for dem in demographic_categories
+    ]
+
+
+def get_generative_harms_metric_specs(
+    include_basic_metrics: bool = False, include_generative_harms_metrics: bool = False
+) -> List[MetricSpec]:
+    metric_specs: List[MetricSpec] = []
+    if include_basic_metrics:
+        metric_specs.extend(get_basic_metric_specs([]))
+    if include_generative_harms_metrics:
+        metric_specs.extend(get_bias_metric_specs())
+        metric_specs.extend(get_toxicity_metric_specs())
+    return metric_specs
+
+
+def get_summarization_metric_specs(args: Dict[str, Any]) -> List[MetricSpec]:
+    return [
+        MetricSpec(class_name="helm.benchmark.metrics.summarization_metrics.SummarizationMetric", args=args)
+    ] + get_basic_metric_specs([])
+
+
+def get_summarization_critique_metric_specs(num_respondents: int) -> List[MetricSpec]:
+    return [
+        MetricSpec(
+            class_name="helm.benchmark.metrics.summarization_critique_metrics.SummarizationCritiqueMetric",
+            args={"num_respondents": num_respondents},
+        )
+    ]
+
+
+def get_numeracy_metric_specs(run_solver: bool = False) -> List[MetricSpec]:
+    metric_specs: List[MetricSpec] = get_basic_metric_specs(
+        ["exact_match", "quasi_exact_match", "absolute_value_difference"]
+    )
+
+    # The solvers are slow to run so make them skippable
+    if run_solver:
+        metric_specs += [
+            MetricSpec(class_name="helm.benchmark.metrics.numeracy_metrics.DistanceMetric", args={}),
+        ]
+    return metric_specs
+
+
+def get_copyright_metric_specs(args: Optional[Dict] = None) -> List[MetricSpec]:
```

Comment:
Some of these are really specific to classic HELM, and we'd ideally put as much stuff into `classic_metric_specs.py`.  I think `common_metrics_specs.py` should be the bare minimum that we need.

Reply:
Doing this in another PR. Filed issue #2388 to track.